### PR TITLE
Show job status in job detail page header

### PR DESCRIPTION
**Pull request by philipnrmn**
This PR introduces a status field to the jobs breadcrumbs, allowing an arbitrary job status to be shown. 

The jobs page uses this to show the status of the longest running task associated with a job. This is the same behaviour as the earlier jobs detail page header. 

son et lumier

### CODE REVIEW

File: src/js/pages/jobs/JobDetailPage.js [hunk 1]
```diff
@@ -240,6 +240,24 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
...
+      longestRunningTask = longestRunningActiveRun.getTasks()
+        .getLongestRunningTask();
+    }
+
+    if (longestRunningTask == null) {
+      return null;
+    }
+
+    const status = TaskStates[longestRunningTask.getStatus()];
+    return status.displayName;
```
Comment (nLight) [inline src/js/pages/jobs/JobDetailPage.js]: http://eslint.org/docs/rules/newline-before-return 🚎  :)

**Comment by nLight** [comment]
retest this please

**Comment by nLight** [comment]
run unit tests

**Comment by nLight** [comment]
Fixed the #nit, rebased, merging

File: src/js/pages/jobs/JobDetailPage.js (hunk 1 above)
Comment (mlunoe) [inline src/js/pages/jobs/JobDetailPage.js]: Put up a PR here: https://github.com/dcos/dcos-ui/pull/1692

**Pull request merged**